kong/pdk/node.lua

```diff
@@ -272,6 +272,9 @@ local function new(self)
     -- 1. user provided node id
     local configuration_node_id = self and self.configuration and self.configuration.node_id
     if configuration_node_id then
+      ngx.log(ngx.WARN, "The `node_id`, which has been deprecated since 3.9, is not recommended to be specified manually!!!\n",
```

Comment:
how about:
`manually specifying a node_id is deprecated as of 3.9, it is not recommended to be specified manually. please make sure it is unique across the cluster. Please note that it will be removed in 4.x`